.github/workflows/build-notifier.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Notification (Daily)
+
+on:
+  workflow_run:
+    workflows: ["Build (Daily)", "Publish to Docker Hub (Daily)"]
+    types:
+      - completed
+
+jobs:
+  discord-notify:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+
+    name: Notify about new daily build
+
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@master
+        with:
+          ref: "${{ github.ref }}"
+          submodules: recursive
+          fetch-depth: 0 # This is set to download the full git history for the repo
+
+      - name: Get current version
+        id: release_info
+        uses: revam/gh-action-get-tag-and-version@v1
+        with:
+          branch: true
+          prefix: v
+          prefixRegex: "[vV]?"
+
+      - id: commit_info
+        name: Shorten Commit Hash for Sentry Release
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const sha = context.sha.substring(0, 7);
+            core.setOutput("sha", sha);
+            
+      - name: Notify discord
+        uses: sarisia/actions-status-discord@v1
```

Comment:
Can we use a discord bot/webhook that allow us to send plain messages (so notnembeds, but also with support for markdown), and have a few buttons (i know bots can add those) that link to our downloads page, github page, and/or docker, etc.?

Reply:
If you can find one. This one does markdown and we can add links.
Just no buttons. AFAIK, all webhooks are embeds. We would need a bot for what you want

Reply:
For now let's leave ifnoff the table. Maybe in a future revision to how we push the updates.